src/libcrun/container.c

```diff
@@ -1548,7 +1548,15 @@ container_delete_internal (libcrun_context_t *context, runtime_spec_schema_confi
       if (UNLIKELY (ret < 0))
         return ret;
       if (ret == 1)
-        return crun_make_error (err, 0, "the container `%s` is not in 'stopped' state", id);
+        {
+
+          ret = libcrun_status_has_read_exec_fifo (state_root, id, err);
+          if (ret <= 0)
+            return crun_make_error (err, 0, "the container `%s` is not in `stopped` state", id);
```

Comment:
Should the error text now say
```
... is not in `stopped` or `created` state
```
?

Reply:
yes, thanks for spotting it.  I'll push a new version